### PR TITLE
feat: align with cairo 0.8.1

### PR DESCRIPTION
**Pull request by CHr15F0x**
Fixes in the sequencer client part.

### CODE REVIEW

**Comment by CHr15F0x** [comment]
Looking at migration..

**Review by kkovaacs — APPROVED** [review]
LGTM

**Review by koivunej — APPROVED** [review]

File: crates/pathfinder/src/core.rs [hunk 1]
```diff
@@ -28,6 +28,10 @@ pub struct ContractHash(pub StarkHash);
 #[derive(Debug, Copy, Clone, PartialEq, Serialize, Deserialize)]
 pub struct ContractStateHash(pub StarkHash);
 
+/// A StarkNet contract's class hash.
+#[derive(Debug, Copy, Clone, PartialEq, Deserialize, Serialize)]
+pub struct ContractClassHash(pub StarkHash);
```
Comment (Mirko-von-Leipzig) [inline crates/pathfinder/src/core.rs:33]: Is this different to `ContractHash`? I would expect they're the same thing; although this naming is more accurate now.

**Review by Mirko-von-Leipzig — APPROVED** [review]
Just the question about possibly duplicating `ContractHash`.

File: crates/pathfinder/src/core.rs (hunk 1 above)
Comment (CHr15F0x) [reply crates/pathfinder/src/core.rs]: I've read the [article again](https://starkware.notion.site/Contracts-As-Classes-641060360aef4d048f7cb172afd57866) and yep, you are right, thanks for spotting :+1:

**Pull request merged**